lms/djangoapps/edraak_university/migrations/0003_universityid_can_edit.py

```diff
@@ -1,19 +0,0 @@
-# -*- coding: utf-8 -*-
-from __future__ import unicode_literals
-
-from django.db import migrations, models
-
-
-class Migration(migrations.Migration):
-
-    dependencies = [
-        ('edraak_university', '0002_university_id_setting'),
-    ]
-
-    operations = [
-        migrations.AddField(
-            model_name='universityid',
-            name='can_edit',
-            field=models.BooleanField(default=True),
-        ),
-    ]
```

Comment:
It was giving unknown field at my side so I had to delete an redo migrations. I was not supposed to push this as a change.